jsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+        "baseUrl": ".",
+        "paths": {
+            "@/components/*": [
```

Comment:
I think the common style is to use `@component` without the `/`? 

Which would create the path `import Link from "@components/Link";

Reply:
If you want, I can make that change and push. Please let me know if I should make it.

Reply:
Got it, changes done. Thanks.

Reply:
Conflicts also resolved.

Reply:
Additional conflicts resolved.

Reply:
Thank you 👍